website/static/css/custom.css

```diff
@@ -218,3 +218,12 @@ input#search_input_react
 input#search_input_react:focus, input#search_input_react:active {
     color: #555;
 }
+
+#panel
+{
+	cursor:auto;
+}
+
+#panel, .flip {
+    cursor: pointer;
```

Comment:
If #panel cursor type is defined in the previous tag then why it is redefined ?

Reply:
Sorry, It was just for testing